packages/rule-babel-config/tests/tests.ts

```diff
@@ -0,0 +1,33 @@
+import * as path from 'path';
+
+import * as ruleRunner from 'sonarwhal/dist/tests/helpers/rule-runner';
+import { RuleLocalTest } from 'sonarwhal/dist/tests/helpers/rule-test-type';
+
+const ruleName = 'babel-config/is-valid';
+
+const tests: Array<RuleLocalTest> = [
+    {
+        name: 'Valid .babelrc configuration should pass',
+        path: path.join(__dirname, 'fixtures', 'valid-schema', '.babelrc')
+    },
+    {
+        name: `Invalid .babelrc configuration should fail`,
+        path: path.join(__dirname, 'fixtures', 'invalid-schema', '.babelrc'),
+        reports: [{ message: `'moduleId' should be string.`}]
+    },
+    {
+        name: 'If package.json doesn\'t contain "babel" property, it should pass',
```

Comment:
shouldn't this be an error? if there is no `.babelrc` and package.json doesn't contain the `babel` property then there is no babel config and you should report an error. Maybe the name is incorrect? (and the file name)

Also, use \` for the comments so you can write doesn't instead of doesn\\'t

Reply:
@sarvaje Good catch! By the way, I think reporting this error is missing in the typescript rule as well.

Reply:
ahhhh it is ok!!
This rules doesn't check if it exists or not, just if it is valid.